src/types/Todo.ts

```diff
@@ -0,0 +1,12 @@
+export interface Todo {
+  user: {
+    id: number;
+    name: string;
+    username: string;
+    email: string;
+  } | null;
```

Comment:
```suggestion
  user: User | null;
```
import the User type